lib/hci-socket/bindings.js

```diff
@@ -392,7 +392,10 @@ NobleBindings.prototype.notify = function(peripheralUuid, serviceUuid, character
   var handle = this._handles[peripheralUuid];
   var gatt = this._gatts[handle];
 
-  if (gatt) {
+  if (handle == 0x000f) {
```

Comment:
I'm not too keen on merging this, this change is super specific to your device, other devices could use handle 0x0f and not have this issue.